markdown_writer.py

```diff
@@ -179,54 +182,61 @@ def write_overall_metrics_tables(
     labels,
     columns,
     file,
+    hide_label_metrics,
 ):
     """Write the overall metrics tables to the markdown file."""
-    file.write("| Metric | Average | Median | 90th percentile |\n")
-    file.write("| --- | --- | --- | ---: |\n")
-    if "Time to first response" in columns:
-        if stats_time_to_first_response is not None:
-            file.write(
-                f"| Time to first response "
-                f"| {stats_time_to_first_response['avg']} "
-                f"| {stats_time_to_first_response['med']} "
-                f"| {stats_time_to_first_response['90p']} |\n"
-            )
-        else:
-            file.write("| Time to first response | None | None | None |\n")
-    if "Time to close" in columns:
-        if stats_time_to_close is not None:
-            file.write(
-                f"| Time to close "
-                f"| {stats_time_to_close['avg']} "
-                f"| {stats_time_to_close['med']} "
-                f"| {stats_time_to_close['90p']} |\n"
-            )
-        else:
-            file.write("| Time to close | None | None | None |\n")
-    if "Time to answer" in columns:
-        if stats_time_to_answer is not None:
-            file.write(
-                f"| Time to answer "
-                f"| {stats_time_to_answer['avg']} "
-                f"| {stats_time_to_answer['med']} "
-                f"| {stats_time_to_answer['90p']} |\n"
-            )
-        else:
-            file.write("| Time to answer | None | None | None |\n")
-    if labels and stats_time_in_labels:
-        for label in labels:
-            if (
-                f"Time spent in {label}" in columns
-                and label in stats_time_in_labels["avg"]
-            ):
+    if (
+        "Time to first response" in columns
+        or "Time to close" in columns
+        or "Time to answer" in columns
+        or (hide_label_metrics is False and len(labels) > 0)
```

Comment:
I don't fully get why this tests for "is False", even though the config variable can be "set to any value" to hide these metrics? I guess it is somehow tied to the "hide_label_metrics=False" in row 81 but I don't fully get that either :)

Btw is this way of handling ENV config variables a Python way of doing things? (I don't know Python that well)

Reply:
`hide_label_metrics is False` is another way of saying that the variable is not set. That is a python type thing from what I can tell.